File: quasar/Quasar.DataTransfer/Estoques/Responses/EstoqueResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Quasar.DataTransfer.Produtos.Responses;

namespace Quasar.DataTransfer.Estoques.Responses
{
    public class EstoqueResponse
    {
        public int Codigo {get; set;}
        public int Quantidade {get; set;}
        public ProdutoResponse Produto {get; set;}

    }
}

File: quasar-front/src/app/shared/models/responses/estoque.response.ts
import { ProdutoResponse } from 'src/app/shared/models/responses/produto.response';

export class EstoqueResponse {
  public codigo: number;
  public quantidade: number;
  public produto: ProdutoResponse;

  constructor(params: Partial<EstoqueResponse>) {
    this.codigo = params.codigo || 0;
    this.quantidade = params.quantidade || 0;
    this.produto = params.produto || new ProdutoResponse({});
  }
}
